doc/nrf/includes/install_sdk_common_prerequisites.txt

```diff
@@ -7,7 +7,7 @@
         After downloading the nRF Util executable, move it to a directory that is in the system :envvar:`PATH`.
         On macOS and Linux, the downloaded file also needs to be given execute permission by typing `chmod +x nrfutil` or by checking the checkbox in the file properties.
 
-  * In nRF Util, the latest version of the ``device`` command.
+  * In nRF Util, v2.8.8 of the ``device`` command.
```

Comment:
@ns-tolu, updated the nrfutil-device version here as discussed and in the following places: 

- https://github.com/nrfconnect/sdk-nrf/pull/21339/files#diff-cfee57acb324ff2654f7853dd928a11aec5b8df29459d3f8aeb45a4f80f70b0dR404 (Recommended versions)
- https://github.com/nrfconnect/sdk-nrf/pull/21339/files#diff-c95823991d62cdb87ea738bfee217fa6470071f7692b5d3747154ee1df423445R451 ( migration guide)
- https://github.com/nrfconnect/sdk-nrf/pull/21339/files#diff-fe5d2ad1e3e8cfa1ecd541e8cd2725c58b094c7ebec0c08bfc6a8424a5de1d94R220 (release notes)